plugins/declarative/src/voicecallhandler.h

```diff
@@ -29,7 +29,9 @@ class VoiceCallHandler : public QObject
     Q_PROPERTY(bool isRemoteHeld READ isRemoteHeld NOTIFY remoteHeldChanged)
     Q_PROPERTY(VoiceCallModel* childCalls READ childCalls NOTIFY childCallsChanged)
     Q_PROPERTY(VoiceCallHandler* parentCall READ parentCall NOTIFY parentCallChanged)
-
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    Q_MOC_INCLUDE("voicecallmodel.h");
```

Comment:
Same thing as already mentioned. Would be nice to have such fixed without mentioning all separately.